recipes/upstream/google-cloud/gke/first_setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Log into the GKE cluster with the credentials used to create it.
+export KUBECONFIG="./$(cat ./terraform.tfvars | grep -v "#" | grep -i prefix | awk -F= '{print $2}' | tr -d '"' | sed 's/ //g')_kube_config.yml"
+gcloud container clusters get-credentials $(cat ./terraform.tfvars | grep -v "#" | grep -i prefix | awk -F= '{print $2}' | tr -d '"')-cluster --region $(cat ./terraform.tfvars | grep -v "#" | grep -i region | awk -F= '{print $2}' | tr -d '"')
+sleep 10
+
+# Assign the cluster-admin role to the "client" user created thanks to the master_auth parameter (../../../../modules/distribution/gke/main.tf).
+cat <<EOF | kubectl apply -f -
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: client-binding
+subjects:
+- kind: User
+  name: client
+roleRef:
+  kind: ClusterRole
+  name: "cluster-admin"
+  apiGroup: rbac.authorization.k8s.io
+EOF
+# https://cloud.google.com/kubernetes-engine/docs/how-to/api-server-authentication#legacy-auth
+# https://stackoverflow.com/questions/58976517/gke-masterauth-clientcertificate-has-no-permissions-to-access-cluster-resource
+sleep 5
+
+export KUBECONFIG="./$(cat ./terraform.tfvars | grep -v "#" | grep -i prefix | awk -F= '{print $2}' | tr -d '"' | sed 's/ //g')_kube_config.yml" 
+
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+
+# Installation of the NGINX Ingress Controller (helm_release resource has destruction issues).
+kubectl create namespace ingress-nginx || true
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm install nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --set controller.service.type="LoadBalancer"
```

Comment:
I see the ingress install occurs here, any reason we can't do this with a `helm_release` resource with `wait` and `wait_for_jobs`?
https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#wait